### PR TITLE
Improve Catalan translations with gender neutral forms

### PR DESCRIPTION
**Pull request by jmaspons**
As suggested by @maelle (https://github.com/r-lib/pkgdown/pull/2333#discussion_r1376209758), I tried to neutralize the gender forms in the translations

### CODE REVIEW

**Comment by jmaspons** [comment]
Do you want to review it, @llrs ?

**Review by llrs — COMMENTED** [review]
I suppose this is a matter of taste, but I dislike the changes. These include some subtle meaning change in my opinion: doing something vs be something. For instance the case of Authors: author as someone how has created [a work](https://dlc.iec.cat/Results?IdE=0014648&DecEntradaText=autor&AllInfoMorf=False&OperEntrada=0&OperDef=0&OperEx=0&OperSubEntrada=0&OperAreaTematica=0&InfoMorfType=0&OperCatGram=False&AccentSen=False&CurrentPage=0&refineSearch=0&Actualitzacions=False) and someone who [presumably has created](https://dlc.iec.cat/Results?DecEntradaText=autoria&AllInfoMorf=False&OperEntrada=0&OperDef=0&OperEx=0&OperSubEntrada=0&OperAreaTematica=0&InfoMorfType=0&OperCatGram=False&AccentSen=False&CurrentPage=0&refineSearch=0&Actualitzacions=False) it.

**Comment by kaitokidi** [comment]
Maybe "autoria de",  "edició de"... could be a middle ground to keep meaning and be gender neutral

**Comment by hadley** [comment]
Any updates here? I'm happy to merge once y'all give the go ahead.

**Comment by llrs** [comment]
On the linked suggestion the most "voted" answer suggested to keep it as is (not merge the PR).
Other suggested other approaches. I like the suggestion at https://github.com/r-lib/pkgdown/pull/2382#issuecomment-1839541659 even if it is longer but I cannot modify the PR (Not sure if I can).

I could review the PR and suggest edits in line with the comment. Then I think you or any other with repository rights approve them and merge this.

**Comment by hadley** [comment]
@llrs that sounds like a great proposed workflow, thanks!

File: po/R-ca.po [hunk 1]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
...
 
 #: build-home-authors.R:77 render.R:96
 msgid "Authors"
-msgstr "Autors"
+msgstr "Autoria"
 
 #: build-home-authors.R:169
 msgid "author"
-msgstr "autor"
+msgstr "autoria"
```
Comment (llrs) [inline po/R-ca.po:54]: autor -> autoria de

File: po/R-ca.po [hunk 2]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
...
+msgstr "traducció"
 
 #: build-home-citation.R:36
 msgid "Citing %s"
 msgstr "Citant %s"
 
 #: build-home-citation.R:91
 msgid "Authors and Citation"
-msgstr "Autors i citació"
+msgstr "Autoria i citació"
```
Comment (llrs) [inline po/R-ca.po:102]: autoria i citació -> autoria i citació de

File: po/R-ca.po [hunk 3]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
...
 
 #: build-home-authors.R:177
 msgid "reviewer"
-msgstr "revisor"
+msgstr "revisió"
 
 #: build-home-authors.R:178
 msgid "thesis advisor"
-msgstr "supervisor de tesi"
+msgstr "supervisió de tesi"
```
Comment (llrs) [inline po/R-ca.po:90]: supervisor de tesi -> supervisió de tesi per

File: po/R-ca.po [hunk 4]
```diff
@@ -234,7 +235,7 @@ msgstr "Nota"
 
 #: rd-data.R:57
 msgid "Author"
-msgstr "Autor"
+msgstr "Autoria"
```
Comment (llrs) [inline po/R-ca.po:238]: autor -> autoria de

File: po/R-ca.po [hunk 5]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
...
 
 #: build-home-authors.R:178
 msgid "thesis advisor"
-msgstr "supervisor de tesi"
+msgstr "supervisió de tesi"
 
 #: build-home-authors.R:179
 msgid "translator"
-msgstr "traductor"
+msgstr "traducció"
```
Comment (llrs) [inline po/R-ca.po:94]: traductor -> traducció de

File: po/R-ca.po [hunk 6]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
...
 
 #: build-home-authors.R:176
 msgid "funder"
-msgstr "finançador"
+msgstr "finançament"
 
 #: build-home-authors.R:177
 msgid "reviewer"
-msgstr "revisor"
+msgstr "revisió"
```
Comment (llrs) [inline po/R-ca.po:86]: revisor -> revisió de

File: po/R-ca.po [hunk 7]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
...
 
 #: build-home-authors.R:175
 msgid "data contributor"
-msgstr "col·laborador de dades"
+msgstr "col·laboració de dades"
 
 #: build-home-authors.R:176
 msgid "funder"
-msgstr "finançador"
+msgstr "finançament"
```
Comment (llrs) [inline po/R-ca.po:82]: finançador -> finançament de

File: po/R-ca.po [hunk 8]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
...
+msgstr "col·laboració"
 
 #: build-home-authors.R:173
 msgid "copyright holder"
 msgstr "titular dels drets d'autor"
 
 #: build-home-authors.R:174
 msgid "maintainer"
-msgstr "mantenidor"
+msgstr "manteniment"
```
Comment (llrs) [inline po/R-ca.po:74]: mantenidor -> manteniment per

File: po/R-ca.po [hunk 9]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
...
+msgstr "compilació"
 
 #: build-home-authors.R:171
 msgid "contractor"
 msgstr "contractista"
 
 #: build-home-authors.R:172
 msgid "contributor"
-msgstr "col·laborador"
+msgstr "col·laboració"
```
Comment (llrs) [inline po/R-ca.po:66]: col·laborador -> col·laboració de

File: po/R-ca.po [hunk 10]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
...
 
 #: build-home-authors.R:169
 msgid "author"
-msgstr "autor"
+msgstr "autoria"
 
 #: build-home-authors.R:170
 msgid "compiler"
-msgstr "compilador"
+msgstr "compilació"
```
Comment (llrs) [inline po/R-ca.po:58]: compilador -> compilació de

File: po/R-ca.po [hunk 11]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
 
 #: build-home-authors.R:72
 msgid "Developers"
-msgstr "Desenvolupadors"
+msgstr "Equip de desenvolupament"
 
 #: build-home-authors.R:77 render.R:96
 msgid "Authors"
-msgstr "Autors"
+msgstr "Autoria"
```
Comment (llrs) [inline po/R-ca.po:50]: Autors -> Autoria de

File: po/R-ca.po [hunk 12]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
 
 #: build-home-authors.R:72
 msgid "Developers"
-msgstr "Desenvolupadors"
+msgstr "Equip de desenvolupament"
```
Comment (llrs) [inline po/R-ca.po:46]: Desenvolupadors -> Desenvolupament de

File: po/R-ca.po [hunk 13]
```diff
@@ -42,63 +43,63 @@ msgstr "Més sobre els autors..."
...
 
 #: build-home-authors.R:174
 msgid "maintainer"
-msgstr "mantenidor"
+msgstr "manteniment"
 
 #: build-home-authors.R:175
 msgid "data contributor"
-msgstr "col·laborador de dades"
+msgstr "col·laboració de dades"
```
Comment (llrs) [inline po/R-ca.po:78]: col·laborador de dades -> col·laboració de dades de

**Review by llrs — COMMENTED** [review]
I couldn't make the suggestions to be accepted automatically by you @hadley. I wrote the old form -> new form.

**Comment by jmaspons** [comment]
Hi! How the proposals differ from the PR, @llrs ? I see some cases which are the same.  Perhaps you can make PR to https://github.com/jmaspons/pkgdown/tree/gender_neutral_catalan to make it easier to understand the changes?

**Comment by llrs** [comment]
I think it would be helpful for @hadley to know which consensus we reached in  Catalan before opening a new PR. As you are the one that requested my review and further feedback from the community of Catalan translators feel free to decide what to do.

**Comment by jmaspons** [comment]
Ah ok, now I understand your proposal, @llrs. I like it for cases where the strings are the section title of the right column of the main page, such as `Desevolupament de` (see https://pkgdown.r-lib.org/). For other cases, it doesn't work because the strings are attributes of the authors. In this case, `autoria de` is wrong.

If there is no consensus, we can leave the translation as is.

**Comment by hadley** [comment]
Sounds like we should leave it as is, so I'm closing the PR. Please let me know if I've misunderstood!

**Pull request closed without merging**